### PR TITLE
Profile desktop team checking please

### PR DESCRIPTION
**Pull request by JyAjAn**

### CODE REVIEW

**Comment by JyAjAn** [comment]
modification made thx ( TA )

**Review by NicosMQ — APPROVED** [review]

File: client/src/Router.jsx [hunk 1]
```diff
@@ -52,7 +53,7 @@ const router = createBrowserRouter([
         element: <Product />,
       },
       {
-        path: "/profile",
+        path: "/profile/:name/:id",
```
Comment (VinchC) [inline client/src/Router.jsx]: choose :id or :name, not both

File: client/src/pages/Profile/Profile.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
+@media (max-width: 600px) {
+  .Larger {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css:3]: larger

File: client/src/pages/Profile/Profile.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+  #ButtonStar {
+    margin: 0 2px;
+    font-size: 1.5rem;
+    color: white;
+  }
+}
+
+@media (max-width: 400px) {
+  .Larger {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css:16]: cf. supra

File: client/src/pages/Profile/Profile.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    font-size: 1rem;
+  }
+
+  #ButtonStar {
+    margin: 0 1px;
+    font-size: 1.2rem;
+  }
+}
+
+.Five-Rate-Active {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css:27]: camelCase

File: client/src/pages/Profile/Profile.css [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.Larger {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css:35]: cf. supra

File: client/src/pages/Profile/Profile.css [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+    color: white;
+  }
+}
+
+@media (max-width: 400px) {
+  .Larger {
+    font-size: 1rem;
+  }
+
+  #ButtonStar {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css]: camelCase

File: client/src/pages/Profile/Profile.css [hunk 7]
```diff
@@ -0,0 +1,111 @@
+@media (max-width: 600px) {
+  .Larger {
+    font-size: 1.2rem;
+  }
+
+  #ButtonStar {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css]: camelCase

File: client/src/pages/Profile/Profile.css [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.rate-value-full::before,
+.rate-value-half::before,
+.rate-value-empty::before {
+  content: "★";
+  color: gold;
+}
+
+.Cate-value-empty::before {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css:65]: camelCase

File: client/src/pages/Profile/Profile.css [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+  color: lightgray;
+}
+
+#ButtonStar {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css]: camelCase

File: client/src/pages/Profile/Profile.css [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+  font-family: var(--font);
+  background-color: #00c3e3;
+  padding: 10px 20px;
+  border-radius: 17px;
+  cursor: pointer;
+  margin-top: 10px;
+}
+
+.Custom-button:hover {
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.css:76]: camelCase

File: client/src/pages/Profile/Profile.jsx [hunk 11]
```diff
@@ -1,8 +1,89 @@
...
+  };
+
+  return (
+    <>
+      <div className="profileContainer">
+        <div className="profileHeader">
+          <img src={user.picture} alt="Profile" className="profileImg" />
+          <div className="profileDetails">
+            <h2>{user.pseudo}</h2>
+            <div className="Five-Rate-Active Larger">
```
Comment (VinchC) [inline client/src/pages/Profile/Profile.jsx:48]: camelCase

**Review by VinchC — CHANGES_REQUESTED** [review]

**Review by VinchC — APPROVED** [review]

**Pull request merged**